json-rpc/src/util.rs

```diff
@@ -177,3 +181,33 @@ pub fn script_view_from_script(script: &Script) -> ScriptView {
 
     view
 }
+
+pub fn script_view_from_script_function(script: &ScriptFunction) -> ScriptView {
+    let name = format!(
+        "{}::{}::{}",
+        script.module().address().short_str_lossless(),
+        script.module().name(),
```

Comment:
I think current ScriptView is mapping Script, thus it will not match well with script function,
I suggest we create ScriptFunctionView{ module_id, function, type_arguments, arguments} for ScriptFunction.
And field `script` will not be rendered when it is script function.

Otherwise, I think type == "script_function" and add fields "module", "function" to current ScriptView is better.